app/src/main/java/com/zeapo/pwdstore/crypto/PgpActivity.kt

```diff
@@ -291,8 +310,8 @@ class PgpActivity : AppCompatActivity(), OpenPgpServiceConnection.OnBound {
                                     null
                                 } else {
                                     HoldToShowPasswordTransformation(
-                                        crypto_password_toggle_show,
-                                        Runnable { crypto_password_show.text = entry.password }
+                                            crypto_password_toggle_show,
```

Comment:
Get rid of all these formatting changes

Reply:
Use the spotless indentation or the ide config ?